tools/tm-bench/main.go

```diff
@@ -12,6 +12,8 @@ import (
 
 	"github.com/tendermint/tendermint/libs/log"
 	tmrpc "github.com/tendermint/tendermint/rpc/client"
+	"os/signal"
+	"syscall"
```

Comment:
Import order is as follows:

``` go
import (
	// stdlib
	
	// external packages
	
	// internal packages
)

```

Reply:
I wonder if we can capture this in the linter ?!

Reply:
I wrote this so many times by now, was wondering the same.